### PR TITLE
Remplace client RNA by association client

### PR DESCRIPTION
**Pull request by yoanficadiere**
* Remplace client RNA to use api-asso 
* Display more information about association

Screenshot 2023-02-08 at 15 59 42

### CODE REVIEW

File: components/association-section/index.tsx [hunk 1]
```diff
@@ -17,18 +17,52 @@ const AssociationSection: React.FC<{
...
+      'Date de publication au journal officiel',
+      formatDate(datePublicationJournalOfficiel),
+    ],
+    ['Date de création', formatDate(dateCreation)],
+    ...(dateDissolution && dateDissolution > dateCreation
+      ? [['Date dissolution', formatDate(dateDissolution)]]
+      : []),
+    ['Adresse du siège', adresseSiege],
+    ['Adresse de gestion', adresseGestion],
+    ['Téléphone', telephone],
```
Comment (XavierJp) [inline components/association-section/index.tsx:51]: Phone, mail and siteweb should use `<a>`with mailto when relevant

File: components/association-section/index.tsx [hunk 2]
```diff
@@ -17,18 +17,52 @@ const AssociationSection: React.FC<{
...
+    ['Email', mail],
+    ['Site web', siteWeb],
+    [
+      'Agrement admininstratif',
+      <ul>
+        {agrement.map((agr) => {
+          return <li>{agr.attributeur}</li>;
+        })}
```
Comment (XavierJp) [inline components/association-section/index.tsx:59]: As in mockup -> would add date and and maybe agrement label

See : https://www.data-asso.fr/annuaire/association/45037469900129

**Review by XavierJp — CHANGES_REQUESTED** [review]
I would update 

```
Pour en savoir plus...
```
with : 
```
Retrouver plus d'informations sur la fiche de cette association (lien vers la fiche) (comptes, effectifs et documents) sur data-asso.

**Comment by XavierJp** [comment]
As soon as https://github.com/etalab/annuaire-entreprises-api-proxy/pull/9 gets deployed, we will be able to use the following route : 
`https://rncs-proxy.api.gouv.fr/association/:id_rna`

**Comment by XavierJp** [comment]
NB : il va falloir également mettre a jour : 
- le changelog
- la page RNA sur le MI et expliquer d'ou vient l'API / comment la retrouver sur https://annuaire-entreprises.data.gouv.fr/sources-de-donnees/m

D'ailleurs sont-ce les données du MI ou une autre administrations ?

File: clients/routes.ts [hunk 3]
```diff
@@ -72,6 +72,7 @@ const routes = {
       },
     },
   },
+  association: 'https://siva-integ1.cegedim.cloud/apim/api-asso/api/structure/',
   rna: {
```
Comment (XavierJp) [inline clients/routes.ts:76]: to clean

File: models/association.ts [hunk 4]
```diff
@@ -59,7 +59,7 @@ const verifyAdressConsistency = async (association: IAssociation) => {
   try {
     const adressInsee = escapeTerm(association.siege.adresse.toLowerCase());
     const adressAssociation = escapeTerm(
-      (association.association.adresse || '').toLowerCase()
+      (association.association.adresseSiege || '').toLowerCase()
     );
```
[the remark this answers is not in the document]
Comment (yoanficadiere) [reply models/association.ts:63]: It's look like working as expected

**Review by XavierJp — APPROVED** [review]
Great. Still a few fixes required but almost here

File: components/header/index.tsx [hunk 5]
```diff
@@ -29,6 +29,14 @@ export const Header: React.FC<IProps> = ({
 }) => (
   <>
     <header role="banner" className="fr-header">
+      <div
```
[the remark this answers is not in the document]
Comment (yoanficadiere) [reply components/header/index.tsx]: Maybe just formatting

File: components/association-section/index.tsx [hunk 6]
```diff
@@ -17,18 +17,69 @@ const AssociationSection: React.FC<{
...
   } = uniteLegale;
 
   const data = [
     ['N° RNA (identifiant d’association)', formatIntFr(idAssociation)],
     ['Nom', nomComplet],
+    ['Famille', libelleFamille],
     ['Objet', objet],
-    ['Adresse', adresse],
+    [
+      'Forme juridique',
```
[the remark this answers is not in the document]
Comment (yoanficadiere) [reply components/association-section/index.tsx:43]: i removed the line regime is undefined

**Pull request merged**